docs/README.md

```diff
@@ -441,13 +441,42 @@ messages.
 
 ### Can I test the pull request in my fork before submitting?
 
-Yes, you can do it.
-
-1. Ensure the `main` branch in your fork is updated with the latest changes.
-2. Create a GitHub [personal access token][pat] (PAT) and add it as an
+Yes, you can do it. 
+
+1. Create a fork of this repository, or if you previously did this, ensure the `main` branch in your fork is updated with the latest changes.
+1. If one does not already exist, create an empty ```gh-pages``` branch in your fork.
+   - for example see: [stackoverflow - how to create an empty branch](https://stackoverflow.com/questions/34100048/create-empty-branch-on-github)
+1. If an OWNERS file has not yet been created for your chart, create and merge a branch into the main branch of your fork to add it.
+   - The file is added as ```charts/partners/<vendor>/<chart-name>/OWNERS```
+   - For example (note the values for ```<chart-name>``` and ```<vendor>``` must match the directory structure) :
+      ```
+      chart:
+         name: <chart-name>
+         shortDescription: <chart description>
+         publicPgpKey: null
+      users:
+         - githubUsername: <your github id>
+      vendor:
+         label: <vendor>
+         name: <decriptive name>
+      ```
+   - Note: the pull request will fail and you will need to manually merge it.
```

Comment:
Before this step .. we also need to enable the github actions on the fork. I think for fork creation, it is disabled by default.

Reply:
Added a note for this.